stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -16,5 +22,76 @@ protected String onGetNodeName(View element) {
         StringUtil.removePrefix(className, "android.view.",
         StringUtil.removePrefix(className, "android.widget."));
   }
+
+  @Override
+  protected void onCopyAttributes(View element, AttributeAccumulator attributes) {
+    String id = getIdAttribute(element);
+    if (id != null) {
+      attributes.add(ID_ATTRIBUTE_NAME, id);
+    }
+  }
+
+  // Copied from Resources.resourceHasPackage because it's marked as @hide
+  private static boolean resourceHasPackage(int resid) {
+    return (resid >>> 24) != 0;
+  }
+
+  @Nullable
+  private static String getIdAttribute(View element) {
+    // Adapted from View.toString()
+
+    int id = element.getId();
+    if (id == View.NO_ID) {
+      return null;
+    }
+
+    String idString = null;
+
+    if (resourceHasPackage(id)) {
+      Resources r = element.getResources();
+      if (r != null) {
+        try {
+          String prefix;
+          String prefixSeparator;
+          switch (id & 0xff000000) {
+            case 0x7f000000:
+              prefix = "";
+              prefixSeparator = "";
+              break;
+            case 0x01000000:
+              prefix = "android";
```

Comment:
This special case statement is probably not needed.  I bet `r.getResourcePackageName` will yield android.